presto-main/src/main/java/com/facebook/presto/sql/planner/planPrinter/PlanPrinter.java

```diff
@@ -478,7 +478,7 @@ public Void visitJoin(JoinNode node, Void context)
             else {
                 nodeOutput = addNode(node,
                         node.getType().getJoinLabel(),
-                        format("[%s]%s", Joiner.on(" AND ").join(joinExpressions), formatHash(node.getLeftHashVariable(), node.getRightHashVariable())));
+                        format("[%s]%s %s", Joiner.on(" AND ").join(joinExpressions), formatHash(node.getLeftHashVariable(), node.getRightHashVariable()), node.isMergeJoinEnabled() ? "[MergeJoinEnabled]" : ""));
```

Comment:
Maybe change this to PreferMergeJoin as it cannot be enabled in execution yet in the java step.